modules/openapi-generator/src/main/resources/typescript-fetch/apis.mustache

```diff
@@ -113,8 +113,8 @@ export class {{classname}} extends runtime.BaseAPI {
         }
         {{/isBasicBasic}}
         {{#isBasicBearer}}
-        if (this.configuration && (this.configuration.accessToken || this.configuration.apiKey)) {
-            const token = this.configuration.accessToken || this.configuration.apiKey;
```

Comment:
why don't we want to use apiKey here?

Reply:
I think the `this.configuration.apiKey` is used for` # isApiKey` block (lines 126), not for `# isBasicBearer`.

Reply:
I think that's a fair argument and this PR would make the code easier to read. 

I will mark this as a breaking change because it might affect people relying on `configuration.apiKey` being used as an accessToken.